### ISSUE TITLE
Keyboard navigation not working for pagination

### ISSUE THREAD

**Issue by KHReinhardt** [fiduswriter/simple-datatables#312]
Hello,
without an href attribute in <a> tags, the pagination is not accessible via keyboard navigation any more.

Version 7.3.0

**Comment by KHReinhardt** [comment]
Works like a charm. Thanks for a quick solution and great support!
